### PR TITLE
Select field error when update module fixed, tpl link fixed

### PR DESCRIPTION
**Pull request by jcriadop**

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
Hi @jcriadop,

Instead of copy-pasting the functions `uninstallSeurCashOnDelivery()` and `installSeurCashOnDelivery()` in each upgrade, you should always call the functions available in the class Seur (after a change on their visibility).

Best regards

**Comment by jcriadop** [comment]
Hi, feature added, now module works with install-uninstall seur class functions instead of single functions of upgrade files.

Best regards.

File: views/templates/admin/AdminSeur.tpl [hunk 1]
```diff
@@ -1,336 +1,336 @@
...
+{/literal}
+</script>
+{/if}
+		
+			<fieldset>
+				<legend>
+					<img src="{$img_dir|escape:'htmlall':'UTF-8'}/logonew.png" />
+			 	</legend>
+				<div id="downloadmanual-seur">
+					<a id="manual_download" href="/modules/seur/manual/seur_manual.pdf" target="_blank" >
```
Comment (Quetzacoalt91) [inline views/templates/admin/AdminSeur.tpl]: This link won't work if the shop is installed in a subfolder.

**Comment by jcriadop** [comment]
Hi @Quetzacoalt91, fixed relative path on AdminSeur.tpl file.

Best Regards.

**Comment by Quetzacoalt91** [comment]
Hi @jcriadop,

Thanks, your PR has just been merged and the changes will be available from tomorrow.

Best regards

**Comment by ivanson7** [comment]
Hi. i don`t know if they are see the connection problem. I attach the log where they can see.

Warning on line 41 en archivo /var/www/parashop.es/modules/seur/classes/Town.php
[2] SoapClient::SoapClient(): SSL operation failed with code 1. OpenSSL Error messages: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure

Warning on line 41 en archivo /var/www/parashop.es/modules/seur/classes/Town.php
[2] SoapClient::SoapClient(): Failed to enable crypto

Warning on line 41 en archivo /var/www/parashop.es/modules/seur/classes/Town.php
[2] SoapClient::SoapClient(https://ws.seur.com/WSEcatalogoPublicos/servlet/XFireServlet/WSServiciosWebPublicos?wsdl): failed to open stream: operation failed

Warning on line 62 en archivo /var/www/parashop.es/modules/seur/classes/Rate.php
[2] SoapClient::SoapClient(): SSL operation failed with code 1. OpenSSL Error messages: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure

Warning on line 62 en archivo /var/www/parashop.es/modules/seur/classes/Rate.php
[2] SoapClient::SoapClient(): Failed to enable crypto

Warning on line 62 en archivo /var/www/parashop.es/modules/seur/classes/Rate.php
[2] SoapClient::SoapClient(https://ws.seur.com/WSEcatalogoPublicos/servlet/XFireServlet/WSServiciosWebPublicos?wsdl): failed to open stream: operation failed

Warning on line 43 en archivo /var/www/perchashop.es/modules/seur/classes/Expedition.php
[2] SoapClient::SoapClient(): SSL operation failed with code 1. OpenSSL Error messages: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure

Warning on line 43 en archivo /var/www/perchashop.es/modules/seur/classes/Expedition.php
[2] SoapClient::SoapClient(): Failed to enable crypto

Warning on line 43 en archivo /var/www/perchashop.es/modules/seur/classes/Expedition.php
[2] SoapClient::SoapClient(https://ws.seur.com/webseur/services/WSConsultaExpediciones?wsdl): failed to open stream: operation failed

**Pull request merged**